File: C/Listas/Lista 01/ex09.c
//Entre com a base e a altura de um retângulo e mostre os resultados:
//a. Perímetro (Perímetro é igual à soma dos 4 lados)
//b. Área (Área é igual à lado vezes lado)
#include <stdio.h>

int main(void){
    int base, altura;
    printf("Insira a base do seu retângulo: ");
    scanf("%d", &base);
    printf("Insira a altura do seu retângulo: ");
    scanf("%d", &altura);

    printf("\nCom nos valores informados, o perímetro do seu retângulo possui um perímetro de %dm", base*2+altura*2);
    printf("\nA área do seu retângulo será de %dm²", base*altura);
}

File: Java/extra/Lista 01/src/ex09.java
/* Entre com a base e a altura de um retângulo e mostre os resultados:
a. Perímetro (Perímetro é igual à soma dos 4 lados)
b. Área (Área é igual à lado vezes lado) */

import java.util.Scanner;

public class ex09 {
    static int perimetro(int a, int b) {
        return a + b;
    }

    static int area(int a, int b) {
        return a * b;
    }

    public static void main(String[] args) {
        int base, altura;
        try (Scanner leitor = new Scanner(System.in)) {
            System.out.println("Digite o valor da base: ");
            base = leitor.nextInt();
            System.out.println("\nDigite o valor da altura: ");
            altura = leitor.nextInt();
        }

        System.out.format("\nO perímetro do retângulo = %d\nA área do retângulo = %d", perimetro(altura, base),
                area(altura, base));
    }
}
